src/aiortc/contrib/media.py

```diff
@@ -591,10 +708,13 @@ def __setsize(self, track):
         """
         Set video height and width.
         """
+        print("Setting video size")
```

Comment:
please remove all prints and make them logs, in this case would just remove because of log below

Reply:
this function is not used anymore, deleted.